types/storybook-addon-jsx/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "private": true,
+    "dependencies": {
+        "@storybook/addons": "^5.2.4",
+        "@storybook/react": "^5.2.4"
```

Comment:
Shouldn't this be a `devDependency`, since it's only used in the tests? 🤔

Reply:
I tried but no `devDependency` are allowed in DefinitelyTyped packages 🤷‍♂

Reply:
Hmmm too bad, 'cause this is the reason my other PR got a `Perf: Worse` tag I think... 😕
Since now it's a dependency even tho it's no dependency for the types... 😕